src/cmd/flux-ping.c

```diff
@@ -279,6 +288,12 @@ int main (int argc, char *argv[])
         log_msg_exit ("optparse_add_option_table");
     if ((optindex = optparse_parse_args (opts, argc, argv)) < 0)
         exit (1);
+    if (optindex != argc - 1) {
+        optparse_print_usage (opts);
+        exit (1);
+    }
+    if (!(target = strdup (argv[optindex])))
```

Comment:
minor: is `target` leaked? (didn't see a `free (target)` but may have missed it.

Reply:
Oops, yup!